README.md

```diff
@@ -5,103 +5,111 @@
 
 [Augury](https://augury.angular.io/) is a Google Chrome Dev Tools extension for debugging Angular 2 applications. You can install the extension from [Chrome Store](https://chrome.google.com/webstore/detail/augury/elgalmkoelokbchhkhacckoklkejnhcd).
 
-Once the extenion is installed you can test it against the demo application https://augury.angular.io/demo/
 
-![Screenshot of Augury](assets/screenloop.gif)
+## Supported version
 
-## Supported Version
+Augury works with application built starting with Angular 2.0.
 
-Currently works with applications built in [Angular 2.0.0](https://github.com/angular/angular/blob/master/CHANGELOG.md#200-2016-09-14) using the Angular Component Router version `3.0.0-beta.2`.
+# Known issues
+
+## Router graph
+
+To be able to view the router graph, you will need to inject the Router in the application _Root_ component as shown below (it must be named `router` exactly).
 
-To view the router graph inject the Router in the application root component as shown below (it must be named `router` exactly).
 ```js
 export default class KitchenSink {
   constructor(private router: Router) {
   }
 }
 ```
-[Example](https://github.com/rangle/augury/blob/dev/example-apps/kitchen-sink-example/source/containers/kitchen-sink.ts)
 
-## Support for AoT (Ahead-Of-Time) Compilation
+[Example code](https://github.com/rangle/augury/blob/dev/example-apps/kitchen-sink-example/source/containers/kitchen-sink.ts#L75)
+
+## Support for AoT (Ahead-Of-Time) compilation
 
 In order for Angular to expose the debug information for AoT applications, you will have to explicitly set the debug flag to `true` in your project's `tsconfig.json` as such:
+
 ```json
 "angularCompilerOptions": {
   /* ... */
   "debug": true
 }
 ```
-Note that this debug flag and `development mode` in Angular runtime are two completely different settings. 
+
+_Note_: This debug flag and `development mode` in Angular runtime are two completely different settings.
```

Comment:
Good point, also won't hurt to mention to have source map if you want to view TypeScript code instead of the compiled/compress JS code.